ocs_ci/utility/aws.py

```diff
@@ -953,6 +970,47 @@ def create_stack(self, s3_url, index, params_list, capabilities):
         logger.info(f"Stackid = {stack_id}")
         return stack_name, stack_id
 
+    def delete_apps_record_set(self):
+        """
+        Delete apps record set that sometimes blocks sg stack deletion.
+            https://github.com/red-hat-storage/ocs-ci/issues/2549
+
+        """
+        cluster_name = config.ENV_DATA['cluster_name']
```

Comment:
How those values get's propagated when we run aws cleanup script? Don't we have the same issue like with stack name that cluster_name is not defined at this point in config?

Reply:
Good point, I forgot to allow passing the cluster_name through as an optional parameter. We can simply pass that during the more general cleanup and pull it from the config during standard cleanup.

Reply:
Done